### PR TITLE
64 feature/friend list/master

### PR DESCRIPTION
**Pull request by V0lk7**
Feature Friend List implemented and working how it was meaned.
Problem about the profile picture not changing when use of refresh button is solved now.

### CODE REVIEW

File: Django_data/apps/FriendsList/templates/RequestList copy.html
Comment (ji2myji2my) [inline Django_data/apps/FriendsList/templates/RequestList copy.html:1]: Delete this file

File: Django_data/apps/FriendsList/views.py [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+            me = request.user
+            all_friend_request = Friend_Request.objects.all().filter(to_user=me)
+            size_request = len(all_friend_request)
+            return render(
+                request, self.template_name,
+                {'me': me, 'all_friend_request': all_friend_request,
+                 'size_request': size_request})
+
+        def post(self, request):
+            logger.info(f"{request.POST = }")
```
Comment (ji2myji2my) [inline Django_data/apps/FriendsList/views.py:65]: Delete debug code

File: Django_data/apps/Twofa/views.py [hunk 2]
```diff
@@ -92,7 +91,7 @@ def get(self, request):
             _failure = "failure_2FA"
             _success = "success_2FA"
         else:
-            logger.info("The fuuuuuuuck ???") 
+            logger.info("The fuuuuuuuck ???")
```
Comment (ji2myji2my) [inline Django_data/apps/Twofa/views.py:94]: Delete debug code

**Review by ji2myji2my — CHANGES_REQUESTED** [review]
Maybe do the transfert .html file from */FriendsList/template to the correct static file front

Kiss

**Review by ji2myji2my — CHANGES_REQUESTED** [review]
Missing a feature

**Review by ji2myji2my — CHANGES_REQUESTED** [review]

File: Django_data/apps/FriendsList/views.py [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+            size_myFriends = len(myFriends)
+            size_people = len(people)
+            return render(
+                request, self.template_name,
+                {'from_user': from_user, 'people': people, 'myFriends': myFriends, 
+                 'size_myFriends': size_myFriends, 'size_people': size_people}
+            )
+
+        def post(self, request):
+            logger.info(f"{request.POST = }")
```
[the remark this answers is not in the document]
Comment (V0lk7) [reply Django_data/apps/FriendsList/views.py:27]: yeah yeah yeah

**Review by DoubleV37 — APPROVED** [review]
Gg great job

**Review by ji2myji2my — APPROVED** [review]
Great job, supreme manager, developer and friend Jean. Always a pleasure to work with you.

**Comment by ji2myji2my** [comment]
Nice jobs team.

**Pull request merged**